src/modules/Game.class.js

```diff
@@ -50,17 +284,52 @@ class Game {
    * `win` - the game is won;
    * `lose` - the game is lost
    */
-  getStatus() {}
+  getStatus() {
+    return this.status;
+  }
+
+  addRandomTile() {
+    const emptyCells = [];
+
+    for (let r = 0; r < 4; r++) {
+      for (let c = 0; c < 4; c++) {
+        if (this.state[r][c] === 0) {
+          emptyCells.push([r, c]);
+        }
+      }
+    }
+
+    if (emptyCells.length > 0) {
+      const randomIndex = Math.floor(Math.random() * emptyCells.length);
+      const [row, col] = emptyCells[randomIndex];
+
+      const newValue = Math.random() < 0.9 ? 2 : 4;
+
+      this.state[row][col] = newValue;
+    }
+  }
 
   /**
    * Starts the game.
    */
-  start() {}
+  start() {
+    this.score = 0;
+    this.status = 'playing';
+
+    // if (this.state.every((row) => row.every((cell) => cell === 0))) {
+    this.addRandomTile(2);
```

Comment:
```suggestion
    this.addRandomTile();
```
and remove redundant comments above and below